source/includes/usage-examples/code-snippets/count.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri = "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
```

Comment:
Can we change this uri declaration to how I did it in [this declaration](https://github.com/mongodb/docs-golang/blob/master/source/includes/usage-examples/code-snippets/insertOne.go#L16-L19)?